service/src/main/scala/za/co/absa/logingw/rest/controller/HealthController.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.logingw.rest.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.{ApiResponse, ApiResponses}
+import io.swagger.v3.oas.annotations.tags.{Tag, Tags}
+
+import java.util.concurrent.CompletableFuture
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.{HttpStatus, MediaType}
+import org.springframework.web.bind.annotation._
+import za.co.absa.logingw.rest.service.HealthService
+
+import scala.concurrent.Future
+
+@RestController
+@RequestMapping(Array("/health"))
```

Comment:
If you see a compelling case for it, we can keep it. Weigh in your options.

It is hard to guess if we will truly need it besides the extension options that the built-in and [possibly custom health indicators](https://www.callicoder.com/spring-boot-actuator/#creating-a-custom-health-indicator) (extending `AbstractHealthIndicator`) offer. At this point, it seems unnecessary to me -- it would be easy to bring it back if we find the specific need for it.